issue_11/script.js

```diff
@@ -0,0 +1,70 @@
+function startLoading() {
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loadingImg.alt = 'ローディング画像';
+    document.getElementById('js-loading').appendChild(loadingImg);
+}
+
+function stopLoading() {
+    document.getElementById('js-loading').remove();
+}
```

Comment:
stopというかremoveしているので
```suggestion
function removeLoading() {
    document.getElementById('js-loading').remove();
}
```

Reply:
Thank you for pointing this out.
I think this is certainly more appropriate.
I have corrected it.
https://github.com/yoshikawa-akr/js-lessons/pull/11/commits/0c09f05b806a4eb3376293b9c58ca8910bd2f527